### PR TITLE
Use codepen in examples

### PR DESCRIPTION
**Pull request by oterral**
[Test](http://mf-chsdi3.dev.bgdi.ch/teo_examples/api/examples.html)

[Here](http://codepen.io/geoadmin/collections/popular/) the collections created.

Ive added codepen in iframe but we could also remove the iframe and only let the link to the codepen.

### CODE REVIEW

**Comment by loicgasser** [comment]
I think the idea is pretty awesome

**Comment by gjn** [comment]
@davidoesch What do you think?

I think it makes sense. Allows the user to directly play with the examples.

PR should include the removal of the example code that was replaced by the codepen examples.

**Comment by davidoesch** [comment]
-iframe : rename it to interact with a layer in iframe or similar (iframe explained is in www.geo.admin.ch/iframe which targets as well a simple audience ) 
- integration of examples in api3.geo.admin.ch : I would even simple put a link to codepen - no examples in API3 site itself to make sure that we actually do not have a js api in the future

**Comment by oterral** [comment]
Ready to review

**Comment by loicgasser** [comment]
http://codepen.io/geoadmin/pen/eJqBVV?editors=0010 KML example tooltip doesn't work for me

**Comment by loicgasser** [comment]
http://codepen.io/geoadmin/pen/grYLdY?editors=0010 remove layer doesn't work

**Comment by loicgasser** [comment]
relative link in `projection available` -> broken

http://codepen.io/geoadmin/pen/GZKEam?editors=0010

**Comment by loicgasser** [comment]
For openlayers examples is it wanted to use the 3.6 version link?

**Comment by oterral** [comment]
@loicgasser all fixed

**Comment by oterral** [comment]
Yes because the api use the 3.6.0

File: chsdi/static/doc/source/api/examples.rst [hunk 1]
```diff
@@ -7,26 +7,105 @@
...
+    }, {
+      label: 'WMTS reprojected on-the-fly in LV95 (EPSG:2056)',
+      codepen: 'GZKEam',
+      cat: 2
+    }, {
+      label: 'All available layers as WMTS',
+      codepen: 'MyYYXR',
+      cat: 2 
+    }, {
+      label: 'More OpenLayers 3 Examples on the official website. All OpenLayers 3 examples are supposed to work with the GeoAdmin API.',
```
Comment (loicgasser) [inline chsdi/static/doc/source/api/examples.rst]: I would stick to `More OpenLayers 3 Examples on the official website.` only

Comment (oterral) [inline chsdi/static/doc/source/api/examples.rst]: Agree

**Comment by loicgasser** [comment]
Last question who/how should we update/add codepens? Related to the github account and geoadmin group?

**Comment by loicgasser** [comment]
I fail to find https://mf-chsdi3.int.bgdi.ch/gal_meteo/examples/geoadmin_geojson.html

**Comment by oterral** [comment]
Code pens are on the account `geoadmin`, code in keepass. I'll send a mail with a recap when this will be merged

**Comment by loicgasser** [comment]
> Code pens are on the account geoadmin, code in keepass. I'll send a mail with a recap when this will be merged

some doc in README is possible then?

**Comment by oterral** [comment]
Here http://codepen.io/geoadmin/pen/ZWzXgv

**Comment by oterral** [comment]
Yes good idea

**Comment by loicgasser** [comment]
Does it make sens to include? https://github.com/geoadmin/mf-chsdi3/commit/c76797de6d5bbe6b0a41e6e23b75f338f6c73da8

**Comment by loicgasser** [comment]
ZWzXgv is nowhere to be found on api doc page

**Comment by oterral** [comment]
GeoJSON example added

**Comment by loicgasser** [comment]
Ok once doc added in README you can merge thanks!

**Comment by oterral** [comment]
README added https://github.com/geoadmin/mf-chsdi3/tree/6ee68e19a4388acef15b99752ed79609dae7d5d8/chsdi/static/doc/examples

**Pull request merged**